### PR TITLE
add sync feature to the showcase app section

### PR DESCRIPTION
**Pull request by wei-lee**
## Motivation

Update the showcase apps section to reflect the latest changes on the ionic showcase app.

Also changed the orders of the tabs to have `Cordova` as the first one.

### CODE REVIEW

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 1]
```diff
@@ -40,6 +39,32 @@ For more information about features, see xref:identity-management.adoc[{keycloak
...
+|Sync Application metrics endpoint| metrics
+|Sync App Port|4000
+|Sync App GraphQL Endpoint| graphql
+|Database Server Hostname|*Hostname of the PostgreSQL server*
+|Database Server Port|*Port number of the PostgreSQL server, usually should be 5432*
+|Database Server Username|*Username of the PostgreSQL database connection*
+|Database Server Password|*Password of the PostgreSQL database connection*
+|Database name|*PostgreSQL database name*
+|====
+.. xref:data-sync.adoc#sync-server-binding-keycloak[Protect the {sync-service} app using {keycloak-service}]
```
Comment (finp) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: is this necessary?

File: modules/ROOT/pages/_partials/identity-management-demo.adoc [hunk 2]
```diff
@@ -3,13 +3,26 @@
 = {keycloak-service}
 
 // tag::idm-demo[]
-
 From the showcase app:
 
-. Press the *Authenticate* menu item. A login screen is displayed.
+[tabs]
```
Comment (finp) [inline modules/ROOT/pages/_partials/identity-management-demo.adoc]: should we switch the tabs everywhere (over time)?

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 3]
```diff
@@ -40,6 +39,32 @@ For more information about features, see xref:identity-management.adoc[{keycloak
 
 . To demonstrate {unifiedpush-service}, xref:push-notifications.adoc#provisioning-push-notifications-service[Provision the {unifiedpush-service} service]. For more information about features, see xref:push-notifications.adoc[{unifiedpush-service} Service]
 
+. To demonstrate {sync-service}:
```
Comment (finp) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: NOTE: You can only demonstrate {sync-service} on Cordova.

File: modules/ROOT/pages/_partials/identity-management-demo.adoc (hunk 2 above)
Comment (wei-lee) [reply modules/ROOT/pages/_partials/identity-management-demo.adoc]: yes, I think so

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc (hunk 3 above)
Comment (wei-lee) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: I mentioned it in the features section, should I mention it here too?

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc (hunk 1 above)
Comment (wei-lee) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: yes

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc (hunk 3 above)
Comment (finp) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: I think it's worth it, in case someone jumps into docs at this point

Comment (wei-lee) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: ok

**Pull request merged**